scripts/dev/generate_embeddable_epJSON_schema/generate_embeddable_epJSON_schema.cpp

```diff
@@ -19,7 +19,7 @@ int main( int argc, char const *argv[] )
 	auto const input_json = json::parse( schema_stream );
 	auto const v_cbor = json::to_cbor( input_json );
 
-	printf("	const static std::array< std::uint8_t, %zu > embeddedSchema = {{\n", v_cbor.size() );
+	printf("	static constexpr std::array< std::uint8_t, %zu > embeddedSchema = {{\n", v_cbor.size() );
```

Comment:
Does this have any impact on performance or is it just more appropriate?

Reply:
I don't think it affects performance, however, it can be used in `constexpr` functions now (It likely could have before, but I haven't tested or read to see if that is true)